app/gui/src/model/execution_context.rs

```diff
@@ -303,6 +303,8 @@ pub struct Visualization {
     pub id:             VisualizationId,
     /// Expression that is to be visualized.
     pub expression_id:  ExpressionId,
+    /// Module to evaluate visualization in context of.
+    pub module:         String,
```

Comment:
This should probably be a `QualifiedName`.

Reply:
OK, done in 8f60565a0

Reply:
Alas, @Frizi, it may not have been good idea:

> [`QualifiedName`] always omit the "Main" module name in its path

But I need to construct `local.Project.Main` module name!